src/test/java/LoginTests.java

```diff
@@ -171,19 +160,16 @@ public void SearchSong () throws InterruptedException {
         passwordField.click();
         passwordField.sendKeys("te$t$tudent");
         submitButton.click();
-        Assert.assertTrue(driver.findElement(avatarSelector).isDisplayed());
-        // WebElement songsMenu = driver.findElement(By.className("songs"));
-        // WebElement songsMenu = driver.findElement(songMenuSelector);
-        driver.findElement(searchBarSelector).click();
-        driver.findElement(searchBarSelector).sendKeys("Veggie Straws");
-        //Assert.assertTrue(driver.findElement().isDisplayed());
+        Assert.assertTrue(getDriver().findElement(avatarSelector).isDisplayed());
+        getDriver().findElement(searchBarSelector).click();
+        getDriver().findElement(searchBarSelector).sendKeys("Veggie Straws");
         Thread.sleep(5000);
```

Comment:
**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.**